src/main/java/net/earthcomputer/clientcommands/command/SnakeCommand.java

```diff
@@ -0,0 +1,158 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.brigadier.CommandDispatcher;
+import net.fabricmc.fabric.api.client.command.v1.FabricClientCommandSource;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.DrawableHelper;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.render.GameRenderer;
+import net.minecraft.client.sound.PositionedSoundInstance;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.sound.SoundEvents;
+import net.minecraft.text.TranslatableText;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.Vec3i;
+import org.lwjgl.glfw.GLFW;
+
+import java.util.LinkedList;
+import java.util.ListIterator;
+import java.util.Random;
+
+import static net.fabricmc.fabric.api.client.command.v1.ClientCommandManager.*;
+
+public class SnakeCommand {
+    public static void register(CommandDispatcher<FabricClientCommandSource> dispatcher) {
+        dispatcher.register(literal("csnake")
+                .executes(ctx -> snake(ctx.getSource())));
+    }
+
+    private static int snake(FabricClientCommandSource source) {
+        source.getClient().send(() -> source.getClient().setScreen(new SnakeGameScreen()));
```

Comment:
This `send` invocation is to prevent the screen from being overwritten by the chat screen closing right? Better add a comment to explain that as it's not obvious.